TradeItIosTicketSDK2/TradeItSDK.swift

```diff
@@ -133,6 +140,8 @@ import UIKit
             return
         }
 
+        self._isDeviceJailbroken = TradeItDeviceManager.isDeviceJailBroken()
```

Comment:
Is it not possible to move this to the `get`? It does not rely on any of the parameters passed in